generator/generator.yml

```
@@ -771,3 +771,21 @@ modules:
   hrSWInstalled:
     walk:
       - hrSWInstalled
+
+#
+# Powercom MIBs
+#
+# https://www.upspowercom.com/Card-DA807.jsp
+#
+  powercom:
+    walk:
+      - 1.3.6.1.4.1.935.1.1.1.1
+      - 1.3.6.1.4.1.935.1.1.1.2
+      - 1.3.6.1.4.1.935.1.1.1.3
+      - 1.3.6.1.4.1.935.1.1.1.4
+      - 1.3.6.1.4.1.935.1.1.1.5
+      - 1.3.6.1.4.1.935.1.1.1.6
+      - 1.3.6.1.4.1.935.1.1.1.7
+      - 1.3.6.1.4.1.935.1.1.1.8
+      - 1.3.6.1.4.1.935.1.1.1.9
+      - 1.3.6.1.4.1.935.2.1.2.1
```

Would you please document these walk OIDs with what identifier they belong to?

See the other examples in the generator file.

For example:
```suggestion
      - 1.3.6.1.4.1.935.1.1.1.1 # upsIdentp
      - 1.3.6.1.4.1.935.1.1.1.2
      - 1.3.6.1.4.1.935.1.1.1.3
      - 1.3.6.1.4.1.935.1.1.1.4
      - 1.3.6.1.4.1.935.1.1.1.5
      - 1.3.6.1.4.1.935.1.1.1.6
      - 1.3.6.1.4.1.935.1.1.1.7
      - 1.3.6.1.4.1.935.1.1.1.8
      - 1.3.6.1.4.1.935.1.1.1.9
      - 1.3.6.1.4.1.935.2.1.2.1
```